# Statistical Analysis of High-Throughput Genomic and Transcriptomic Data 
Fall/Herbst-semester 2018

## Lectures
Mondays 9.00-9.45 (Y27-H-46), 10.00-10.45 (Y27-H-46)

## Exercises
Monday 11.00-11.45 (Y01-F-50)

## Lecturers

[Mr. Lukas Weber](https://github.com/lmweber), final year PhD student, [IMLS](http://www.imls.uzh.ch/index.html), UZH  

[Dr. Hubert Rehrauer](http://www.fgcz.ch/the-center/people/rehrauer.html), Group Leader of Genome Informatics at [FGCZ](http://www.fgcz.ch/)  

[Prof. Dr. Mark Robinson](https://robinsonlabuzh.github.io/), Associate Professor of Statistical Genomics, [IMLS](http://www.imls.uzh.ch/index.html), UZH  

[Dr. Charlotte Soneson](http://csoneson.github.io/), Postdoctoral Associate, [IMLS](http://www.imls.uzh.ch/index.html), UZH  


## Schedule

| Date  | Lecturer | Topic | Exercise | JC1 | JC2 |
| --- | --- | --- | --- | --- | --- |
| 17.09.2018  | Mark + Hubert  | admin; mol. bio. basics | R markdown; git(hub) | | |
| 24.09.2018  | Hubert | NGS intro; exploratory data analysis | EDA in R | | |
| 01.10.2018  | Mark + Hubert | interactive technology/statistics session  | group exercise: technology pull request | | |
| 08.10.2018  | Hubert | mapping  | Rsubread | | |
| 15.10.2018  | Hubert | RNA-seq quantification    | RSEM  | | |
| 22.10.2018  | Mark | limma + friends | linear model simulation + design matrices | [Averaged gene expressions for regression](http://www.web.stanford.edu/~hastie/Papers/averaged_regression.pdf) (AS, LB, MK) | [Detection and accurate false discovery rate control of differentially methylated regions from whole genome bisulfite sequencing](https://academic.oup.com/biostatistics/advance-article/doi/10.1093/biostatistics/kxy007/4899074) (DT, HP) |
| 29.10.2018  | Charlotte | hands-on session #1: RNA-seq  | FASTQC/Salmon/etc. | [Capturing Heterogeneity in Gene Expression Studies by Surrogate Variable Analysis](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.0030161) (MS, CR) | X |
| 05.11.2018  | Mark | edgeR+friends 1 | basic edgeR/voom | [Overcoming systematic errors caused by log-transformation of normalized single-cell RNA sequencing data](https://www.biorxiv.org/content/early/2018/08/31/404962) (RB, RG) | [A Universal Algorithm to Detect Rare or Novel Cell Types in High-Throughput Single-Cell Gene Expression Data](https://www.biorxiv.org/content/early/2018/10/13/438382) (JC, TF, MS) |
| 12.11.2018  | Mark | edgeR+friends 2  | GLM/DEXSeq | [A general and flexible method for signal extraction from single-cell RNA-seq data](https://www.nature.com/articles/s41467-017-02554-5) (AL, VL)  | [Integrating single-cell transcriptomic data across different conditions, technologies, and species](https://www.nature.com/articles/nbt.4096) (PV, FN, ES) |
| 19.11.2018  | Mark | single-cell dim. reduction + clustering; FDR  | conquer | [Normalization of RNA-seq data using factor analysis of control genes or samples](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4404308/) (RM, JD, CV) | [Diffusion maps for high-dimensional single-cell analysis of differentiation data](10.1093/bioinformatics/btv325) (SP, GK) |
| 26.11.2018  | Lukas | hands-on session #2: cytometry  | cytof null comparison | [Epigenome-wide association studies without the need for cell-type composition](https://www.ncbi.nlm.nih.gov/pubmed/24464286) (RL, SG) | X |
| 03.12.2018  | Hubert | classification  | MLInterfaces | [Predicting cell types in single cell mass cytometry data](https://www.biorxiv.org/content/early/2018/05/07/316034) (CM, SS) | [Bayesian Trees for Automated Cytometry Data Analysis](https://www.biorxiv.org/content/early/2018/09/19/414904) (CB, XC)|
| 10.12.2018  | Mark | loose ends: HMM, EM, robustness   | segmentation, peak finding | [Differential expression analysis for sequence count data](https://genomebiology.biomedcentral.com/articles/10.1186/gb-2010-11-10-r106) (AA, PS) | [Visualizing Data using t-SNE](https://lvdmaaten.github.io/publications/papers/JMLR_2008.pdf) (MJT, TB, MP)  |    
| 17.12.2018  | Mark | hands-on session #3: single-cell RNA-seq  | full scRNA-seq pipeline |  [Bump hunting to identify differentially methylated regions in epigenetic epidemiology studies](https://www.ncbi.nlm.nih.gov/pubmed/22422453) (SB,ST) | tba (DS, JS, CL) |



## Useful Links
[Simply Statistics blog](https://simplystatistics.org/)  
[Getting Genetics Done blog](http://www.gettinggeneticsdone.com/)  
[Omics Omics blog](http://omicsomics.blogspot.ch/)  

## Course material

Assuming you have git installed locally, you can check out the entire set of course materials with the following command (from command line):
```
git clone https://github.com/sta426hs2018/material.git
```  
Alternatively, for a ZIP file of the repository, you can click on the (green) 'Clone or download' (top right) and then click 'Download ZIP'.
